Sming/Makefile-rboot.mk

```diff
@@ -29,7 +29,7 @@ RBOOT_ROM_0      ?= rom0
 RBOOT_ROM_1      ?= rom1
 RBOOT_SPIFFS_0   ?= 0x100000
 RBOOT_SPIFFS_1   ?= 0x300000
-RBOOT_LD_0 ?= $(SMING_HOME)/compiler/ld/rboot.rom0.ld
+RBOOT_LD_0 ?= rboot.rom0.ld
```

Comment:
Yes it will. It allows you to have a different _MEMORY_ section in your projects linker files and still use all the other settings provided by the framework. I will try to provide an example of a Makefile soon that uses this and and other features to automatically do most of the memory calculations and generate all the required files.